src/android/webview.kt

```diff
@@ -56,6 +71,10 @@ open class WebViewClient (activity: WebViewActivity) : android.webkit.WebViewCli
       return false
     }
 
+    if (url.scheme == "reload") {
```

Comment:
Lets just reuse `ipc://` instead as this introduces possibly another CSP entry.

Lets use

```
ipc://reload
``` 

or something like it

Reply:
This was part of my initial attempt, we don't need it at all. Glad you spotted it.